feature/auth/forgot_password/build.gradle.kts

```diff
@@ -54,6 +54,7 @@ dependencies {
     implementation(project(mapOf("path" to ":data:auth")))
     implementation(project(mapOf("path" to ":core:network")))
     implementation(project(mapOf("path" to ":feature:auth:create_password")))
```

Comment:
Use implementation(project(":feature:auth:create_password)) at the end of file, and remove this line